Makefile

```
@@ -0,0 +1,43 @@
+PERL5LIB_:=../..:os-autoinst:lib:tests/installation:tests/x11:tests/qa_automation:tests/virt_autotest:tests/cpu_bugs:$$PERL5LIB
+
+.PHONY: all
+all:
+
+.PHONY: help
+help:
+	echo "Call 'make test' to call tests"
+
+.PHONY: prepare
+prepare:
+	git clone https://github.com/os-autoinst/os-autoinst.git
+	$(MAKE) check-links
+	cd os-autoinst && cpanm -nq --installdeps .
+	cpanm -nq --installdeps .
```

I think this should be simplified with a new make rule in the os-autoinst Makefile. Something that we would be able to call like:

```suggestion
.PHONY: prepare
prepare:
	git clone --depth 1 https://github.com/os-autoinst/os-autoinst.git
	$(MAKE) -C os-autoinst test-distribution-prepare
```